presentation/src/main/java/com/android254/presentation/login/view/LoginScreen.kt

```diff
@@ -15,20 +15,162 @@
+                                R.drawable.ic_topbar_bg_login
+                            )
+                        },
+                        contentDescription = "background_image",
+                        contentScale = ContentScale.FillBounds
+                    )
+                    LargeTopAppBar(
+                        title = { Text(stringResource(R.string.sign_in_label)) },
+                        navigationIcon = {
+                            IconButton(
+                                onClick = { /* doSomething() */ },
+                            ) {
+                                Icon(
+                                    painter = painterResource(id = R.drawable.ic_back_arrow),
+                                    contentDescription = "Localized description"
+                                )
+                            }
+                        },
+                        scrollBehavior = scrollBehavior,
+                        colors = TopAppBarDefaults.largeTopAppBarColors(
+                            containerColor = Color.Transparent,
+                            titleContentColor = if (darkTheme) Color(0xFFF2F0F4) else MaterialTheme.colorScheme.onPrimary,
+                            navigationIconContentColor = if (darkTheme) Color(0xFFF2F0F4) else MaterialTheme.colorScheme.onPrimary
+                        )
+                    )
+                }
+            }
+        ) { paddingValues ->
+            Column(
+                modifier = Modifier
+                    .padding(paddingValues)
+                    .fillMaxWidth(),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Spacer(modifier = Modifier.height(50.dp))
+                GoogleAuthButton(
+                    onClick = { navController.navigate(Screens.Home.route) },
+                    modifier = Modifier.width(200.dp)
+                ) {}
+                Spacer(modifier = Modifier.height(32.dp))
+                Row(horizontalArrangement = Arrangement.Center) {
+                    Text(text = " - OR - ")
+                }
+                Spacer(modifier = Modifier.height(36.dp))
+                Column(modifier = Modifier.padding(38.dp)) {
+                    DroidConTextField(label = "Email Address")
+                    Spacer(modifier = Modifier.height(24.dp))
+                    DroidConTextField(label = "Password")
+                    Spacer(modifier = Modifier.height(24.dp))
+                    Button(
+                        onClick = { navController.navigate(Screens.Home.route) },
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(48.dp),
+                        shape = RoundedCornerShape(7.dp),
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = if (darkTheme) Color.Black else MaterialTheme.colorScheme.primary,
```

Comment:
This checks are already there in theming unless there's a reason you'd prefer this way

Reply:
Colors on the theme are not consistent with the UI designs so I had to do this check to force it to adhere to the design.